tsconfig.json

```diff
@@ -21,6 +21,6 @@
     "jsx": "react-jsx"
   },
   "include": [
-    "src"
+    "src/routes"
```

Comment:
URGENT: Don't do this!!!
You've stopped any file outside of src/routes from compiling

Change the include to src/**/* to match all files inside src (or something similar I think, contact me if you can't figure it out)